DXRFixes/DeusEx/Classes/Animal.uc

```diff
@@ -0,0 +1,13 @@
+class DXRAnimal merges Animal;
+// TODO: we can change this to injects but it breaks savegame compatibility
```

Comment:
technically we did change rng for skill/aug strengths, but maybe we can wait for a bigger compatibility break